catalog/designers/erikkennedy/bio.html

```diff
@@ -0,0 +1,2 @@
+<p>Erik D. Kennedy is a UX/UI designer who's worked with clients ranging from Fortune 100 to Y-Combinator startups. He's also taught thousands of students around the world via his online school, Learn UI Design. He lives in Seattle, WA and spends his free time doing Brazilian jiu-jitsu and playing Flamenco guitar.</p>
```

Comment:
should we use "who is" instead of informal contraction who's?

Reply:
I guess it would be better yes.